sdk/monitor/test-resources.bicep

```diff
@@ -252,3 +252,5 @@ output INGESTION_DATA_COLLECTION_RULE_IMMUTABLE_ID string = dataCollectionRule.p
 output STORAGE_ACCOUNT_NAME string = storageAccountName
 output STORAGE_ID string = storageAccount.id
 output RESOURCE_ID string = resourceGroup().id
+output WORKSPACE_PRIMARY_RESOURCE_ID string = primaryWorkspace.id
```

Comment:
Should we remove the storage account stuff from this file?

Reply:
We might need it later so I thought we could leave it for now?

Reply:
If we aren't using it we should remove it as it costs money.